src/App.tsx

```diff
@@ -15,71 +15,101 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export type Props = {};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export type State = {
+  selectedGood: string;
+  className: string,
+};
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends Component<Props, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+    className: 'has-background-success-light',
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  selectedGood = (item: string) => {
+    this.setState({ selectedGood: item });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  clearGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        {
+          selectedGood
+            ? (
+              <h1 className="title is-flex is-align-items-center">
+                {selectedGood}
+                {' '}
+                is selected
+                { /* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+                <button
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                  onClick={this.clearGood}
+                />
+              </h1>
+            )
+            : <h1 className="title">No goods selected</h1>
+        }
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {
+              goods.map(good => (
+                <tr
+                  data-cy="Good"
+                  key={good}
+                  className={
+                    selectedGood === good
+                      ? this.state.className
+                      : ''
+                  }
```

Comment:
Better install [classnames](https://www.npmjs.com/package/classnames) and use it instead of the ternary operator.
The ternary operator is very limited in its functionality to be used in classNames. 
[classnames](https://www.npmjs.com/package/classnames) usually used in real projects and would be great to work with it now and would be needed to do in test tasks.

Reply:
okay, I was thinking about it but choose an easier way for me 🫠